From 8dffef568fc2952afab4c9fafadad799f4c56579 Mon Sep 17 00:00:00 2001
From: Laura Lawrence <Laura.Lawrence@freescale.com>
Date: Mon, 19 May 2008 17:32:22 -0500
Subject: [PATCH] ENGR00076811 Config Audio and SDMA buffers in IRAM on i.MX37

Configure Build to place audio and sdma bufs in iram by default on i.MX37

Signed-off-by: Laura Lawrence <Laura.Lawrence@freescale.com>
---
 arch/arm/configs/imx37_3stack_defconfig |   31 +++++++++++++++++++++++++++++--
 1 files changed, 29 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/imx37_3stack_defconfig b/arch/arm/configs/imx37_3stack_defconfig
index 57c8f5e..3443c8b 100644
--- a/arch/arm/configs/imx37_3stack_defconfig
+++ b/arch/arm/configs/imx37_3stack_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.24
-# Mon Mar 31 09:55:31 2008
+# Mon May 19 17:24:16 2008
 #
 CONFIG_ARM=y
 CONFIG_SYS_SUPPORTS_APM_EMULATION=y
@@ -156,18 +156,25 @@ CONFIG_ARCH_MXC=y
 #
 # CONFIG_ARCH_MXC91321 is not set
 CONFIG_ARCH_MX37=y
+# CONFIG_ARCH_MX35 is not set
 # CONFIG_ARCH_MX3 is not set
 # CONFIG_ARCH_MX27 is not set
 # CONFIG_ARCH_MX21 is not set
 CONFIG_I2C_MXC_SELECT1=y
 CONFIG_I2C_MXC_SELECT2=y
 CONFIG_MXC_SDMA_API=y
+CONFIG_SDMA_IRAM=y
+CONFIG_SDMA_IRAM_SIZE=0x1000
 # CONFIG_I2C_MXC_SELECT3 is not set
 
 #
 # MX37 Options
 #
 CONFIG_MACH_MX37_3DS=y
+
+#
+# SDMA options
+#
 CONFIG_ARCH_MXC_HAS_NFC_V3=y
 CONFIG_ARCH_MXC_HAS_NFC_V3_1=y
 
@@ -633,6 +640,13 @@ CONFIG_INPUT_EVDEV=y
 # Input Device Drivers
 #
 CONFIG_INPUT_KEYBOARD=y
+CONFIG_KEYBOARD_ATKBD=y
+# CONFIG_KEYBOARD_SUNKBD is not set
+# CONFIG_KEYBOARD_LKKBD is not set
+# CONFIG_KEYBOARD_XTKBD is not set
+# CONFIG_KEYBOARD_NEWTON is not set
+# CONFIG_KEYBOARD_STOWAWAY is not set
+# CONFIG_KEYBOARD_MXC is not set
 CONFIG_KEYBOARD_MPR084=y
 # CONFIG_INPUT_MOUSE is not set
 # CONFIG_INPUT_JOYSTICK is not set
@@ -655,7 +669,10 @@ CONFIG_TOUCHSCREEN_TSC2007=y
 #
 # Hardware I/O ports
 #
-# CONFIG_SERIO is not set
+CONFIG_SERIO=y
+CONFIG_SERIO_SERPORT=y
+CONFIG_SERIO_LIBPS2=y
+# CONFIG_SERIO_RAW is not set
 # CONFIG_GAMEPORT is not set
 
 #
@@ -880,6 +897,7 @@ CONFIG_FB_MXC=y
 CONFIG_FB_MXC_SYNC_PANEL=y
 CONFIG_FB_MXC_EPSON_VGA_SYNC_PANEL=y
 CONFIG_FB_MXC_TVOUT_TVE=y
+# CONFIG_FB_MXC_CLAA_WVGA_SYNC_PANEL is not set
 # CONFIG_FB_MXC_TVOUT is not set
 # CONFIG_FB_MXC_TVOUT_CH7024 is not set
 # CONFIG_FB_MXC_ASYNC_PANEL is not set
@@ -973,8 +991,11 @@ CONFIG_SND_SOC=y
 #
 CONFIG_SND_MXC_SOC=y
 CONFIG_SND_MXC_SOC_SSI=y
+CONFIG_SND_MXC_SOC_IRAM=y
 CONFIG_SND_SOC_IMX_3STACK_WM8350=y
+# CONFIG_SND_SOC_IMX_3STACK_AK4647 is not set
 CONFIG_SND_SOC_WM8350=y
+# CONFIG_SND_SOC_AK4647 is not set
 
 #
 # Open Sound System
@@ -1028,6 +1049,7 @@ CONFIG_MMC_BLOCK_BOUNCE=y
 # CONFIG_MMC_SPI is not set
 # CONFIG_MMC_MXC is not set
 CONFIG_MMC_IMX_ESDHCI=m
+# CONFIG_MMC_IMX_ESDHCI_PIO_MODE is not set
 # CONFIG_NEW_LEDS is not set
 CONFIG_RTC_LIB=y
 CONFIG_RTC_CLASS=y
@@ -1101,6 +1123,7 @@ CONFIG_MXC_IPU_V3=y
 # MXC PMIC support
 #
 # CONFIG_MXC_SPI_PMIC_CORE is not set
+# CONFIG_MXC_I2C_MCU_PMIC_CORE is not set
 # CONFIG_MXC_PMIC is not set
 
 #
@@ -1131,6 +1154,10 @@ CONFIG_MXC_VPU=y
 # CONFIG_MXC_VPU_DEBUG is not set
 
 #
+# MXC Asynchronous Sample Rate Converter support
+#
+
+#
 # File systems
 #
 CONFIG_EXT2_FS=y
-- 
1.5.4.4

